### PR TITLE
Routes and view components

### PR DESCRIPTION
**Pull request by billionsjoel**
- Create routes and view components (Rockets, Missions, My Profile, Dragons [only if your team has 3 members]). Use <NavLink /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).
- Create directories for all Redux state slice files (rockets, missions, dragons [only if your team has 3 members]).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
     <div className="App">
-      <Header />
+      <Router>
+        <Header />
+        <main>
+          <Routes>
+            <Route path="/" element={<Rocket />} />
+            <Route path="missions" element={<Mission />} />
+            <Route path="my-profile" element={<Profile />} />
+          </Routes>
```
Comment (orozCoding) [inline src/App.js:18]: We can import the Navigate method in the react-router-dom.

Line 2 would look like:

`import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';`

And after line 17 we could add the following code:

`<Route path="*" element={<Navigate to="/" />} />`

This way, any other path won't lead to an empty page, but to the main one (rockets).

Alternatively, we could create a "This page doesn't exist" page, but I believe that is not necessary.

**Review by orozCoding — APPROVED** [review]
Hi, @billionsjoel

Good job! I love the way it's looking.

I believe you completed successfully the task and there's no other required change.



Even so, I would suggest an optional change that could improve our app.

Right now, the user is able to successfully navigate through our pages. But if the user, for some random reason, tries to navigate to a non-existing page, the view component will be showing an empty page.

I suggest we import the Navigate method and create a Route to "*" path that re-directs the user to the home page. See the comments for a better understanding.

Again, great job!

See you later.

File: src/App.js (hunk 1 above)
Comment (billionsjoel) [reply src/App.js]: Hi @orozCoding , i have made those changes. Kindly re-review and  let me know if we can merge. 😄

**Review by orozCoding — APPROVED** [review]
Hi @billionsjoel 

Nice job!

I believe that would be all for now. We are ready to move on!

**Pull request merged**